Products/CMFPlomino/tests/layout.txt

```diff
@@ -0,0 +1,483 @@
+Plomino Layout tests
+=========================
+
+We need to test code for
+ - automatically creating fieldsets
+ - doing labels
+ - handling labels and hidewhens
+ - doing clean up of various non-useful plomino style combinations
+
+
+>>> from lxml import etree, html
+>>> #pphtml = lambda myhtml: etree.tostring(html.fromstring(myhtml.replace('\t','').replace('\n','')), encoding='unicode', pretty_print=True)
+>>> from BeautifulSoup import BeautifulSoup; pphtml = lambda myhtml: BeautifulSoup(myhtml).prettify()
```

Comment:
do we really want to add an extra dependency (BeautifulSoup) just for testing ? up to you, but if we do, we must add it in setup.py [test] section or it will break the tests